firefox-ios/Client/Frontend/PasswordManagement/Cells/LoginDetailTableViewCell.swift

```diff
@@ -62,8 +62,7 @@ class LoginDetailTableViewCell: UITableViewCell,
 
     lazy var descriptionLabel: UITextField = .build { [weak self] label in
         guard let self = self else { return }
-
-        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body, size: UX.descriptionFontSize)
+        label.font = FXFontStyles.Regular.body.scaledFont().withSize(UX.descriptionFontSize)
```

Comment:
thanks for these updates! We can remove the `fontSize` property and rely on `FXFontStyles.Regular.body.scaledFont()` and confirm with @cwzilla on the changes. If we want to following the existing font size `callout` may be better.